### ISSUE TITLE
Thank you so much

### ISSUE THREAD

**Issue by noer180895** [przeor/react-router-flux-starter-kit#4]
thank you so much for your repo i think very helpful me.

**Comment by przeor** [comment]
@noer180895 you are welcome:-) ping me on my email if any questions will occur:-) btw. use this tutorial for grasping more https://egghead.io/series/react-flux-architecture ... this is my inspiration for that repo:-)

**Comment by noer180895** [comment]
@przeor yes thanks. much tutorial about react flux on your link inspiration. btw i think you is very experienced with isomorphic technology, right?

**Comment by przeor** [comment]
plz send me a line, you can find my email on the GH profile

**Comment by alfondotnet** [comment]
Awesome!!!!! This is exactly what I was looking for :+1:
